lib/application.js

```diff
@@ -284,7 +285,10 @@ app.handler = function(type, options) {
   var handler = this._handlers[type] = remotes.handler(type, options);
 
   remotes.classes().forEach(function(sharedClass) {
-    sharedClass.ctor.emit('mounted', app, sharedClass, remotes);
+    /* HACK!!! */
+    if (sharedClass.name !== 'ctrl') {
```

Comment:
@bajtos, I agree, controllers shouldn't really be mounted since they are resolved on a per request basis. However, I'm not aware if there are any other parts of LoopBack that use the "mounted" event such as the SDKs. Thoughts?